.github/workflows/ci.yml

```
@@ -137,13 +136,66 @@ jobs:
+      # Only run if API keys are available (skips on fork PRs)
+      if: ${{ secrets.OPENAI_API_KEY != '' && secrets.ANTHROPIC_API_KEY != '' }}
+      run: |
+        pytest tests/integration/ -v --cov=src/noveum_trace --cov-report=xml --cov-report=term-missing --junitxml=junit.xml
+      env:
+        INTEGRATION_TESTS: "true"
+        OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
+        ANTHROPIC_API_KEY: ${{ secrets.ANTHROPIC_API_KEY }}
+
+    - name: Run mock integration tests (fallback for forks)
+      # Run lightweight tests when API keys are not available
+      if: ${{ secrets.OPENAI_API_KEY == '' || secrets.ANTHROPIC_API_KEY == '' }}
```

actionlint (1.7.7)

context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)